### PR TITLE
[Feature][Flink SQL CDC] Add "MySQL CDC Source Connector"

### PR DESCRIPTION
**Pull request by yuzq24**
1. Add MySQL CDC support for FlinkSQL
    you can see https://github.com/ververica/flink-cdc-connectors

2. I tested the SET configuration and found that the "'" configuration did not take effect. So I removed the "'" of set in the "flink.sql.conf.template" test example.

### CODE REVIEW

**Comment by Hisoka-X** [comment]
Hi @legendtkl PTAL

**Comment by legendtkl** [comment]
Hi, @1141086028 , LGTM with a minor comment.

Dependency check failed with detailed message https://github.com/apache/incubator-seatunnel/runs/7314242930?check_suite_focus=true  

It could be fixed by adding jar `flink-sql-connector-mysql-cdc-2.1.1.jar` into file https://github.com/apache/incubator-seatunnel/blob/dev/tools/dependencies/known-dependencies.txt

Look forward to your more contribute about Flink SQL module ~

Thanks.

**Comment by yuzq24** [comment]
Hi, @legendtkl，I have finished adding. 

I will make as much contribution as possible ~~

**Comment by legendtkl** [comment]
Hi, @1141086028 , the conflicts should be resolved first, and the jars in [known-dependencies.txt](https://github.com/apache/incubator-seatunnel/blob/dev/tools/dependencies/known-dependencies.txt) should be arranged with dictionary order.

**Comment by yuzq24** [comment]
Hi, @legendtkl. I have sorted by dictionary

**Comment by legendtkl** [comment]
Hi, @1141086028 , please check the failed check step "Backend / Dependency licenses" , it seems a missing dependency jar is needed https://github.com/apache/incubator-seatunnel/runs/7382295880?check_suite_focus=true

File: seatunnel-core/seatunnel-core-flink-sql/pom.xml [hunk 1]
```diff
@@ -60,6 +60,16 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.ververica</groupId>
+            <artifactId>flink-sql-connector-mysql-cdc</artifactId>
+            <version>${flink.cdc.version}</version>
+        </dependency>
```
Comment (ashulin) [inline seatunnel-core/seatunnel-core-flink-sql/pom.xml:67]: `flink-sql-connector-*` dependency should be added to `seatunnel-connectors-flink-sql-dist` module

File: seatunnel-examples/seatunnel-flink-sql-examples/pom.xml [hunk 2]
```diff
@@ -95,5 +95,11 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-log4j12</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-runtime-web_${scala.binary.version}</artifactId>
+            <version>${flink.version}</version>
+        </dependency>
```
Comment (ashulin) [inline seatunnel-examples/seatunnel-flink-sql-examples/pom.xml]:
```suggestion
        <dependency>
            <groupId>org.apache.flink</groupId>
            <artifactId>flink-runtime-web_${scala.binary.version}</artifactId>
        </dependency>
```
Same the above.

File: seatunnel-core/seatunnel-core-flink-sql/pom.xml [hunk 3]
```diff
@@ -60,6 +60,16 @@
...
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-runtime-web_${scala.binary.version}</artifactId>
+            <version>${flink.version}</version>
+        </dependency>
```
Comment (ashulin) [inline seatunnel-core/seatunnel-core-flink-sql/pom.xml]:
```suggestion
        <dependency>
            <groupId>org.apache.flink</groupId>
            <artifactId>flink-runtime-web_${scala.binary.version}</artifactId>
        </dependency>
```
nit: The dependencies in `<dependencyManagement>` are referenced here, `<version>` can be omitted

**Review by ashulin — COMMENTED** [review]
I left some comments.

**Comment by 1141086028** [comment]
Hi, @legendtkl, I checked the error log information and found that "org.apache.seatunnel:connector-common:jar:2.1.3-SNAPSHOT" was missing, but I didn't reference this jar. I didn't know how to solve this problem

**Comment by yuzq24** [comment]
> Hi, @1141086028 , please check the failed check step "Backend / Dependency licenses" , it seems a missing dependency jar is needed https://github.com/apache/incubator-seatunnel/runs/7382295880?check_suite_focus=true

Hi, @legendtkl, I checked the error log information and found that "org.apache.seatunnel:connector-common:jar:2.1.3-SNAPSHOT" was missing, but I didn't reference this jar. I didn't know how to solve this problem

**Comment by legendtkl** [comment]
> > Hi, @1141086028 , please check the failed check step "Backend / Dependency licenses" , it seems a missing dependency jar is needed https://github.com/apache/incubator-seatunnel/runs/7382295880?check_suite_focus=true
> 
> Hi, @legendtkl, I checked the error log information and found that "org.apache.seatunnel:connector-common:jar:2.1.3-SNAPSHOT" was missing, but I didn't reference this jar. I didn't know how to solve this problem

Hi, @1141086028 , I checked the error log. The failure is caused by the jar "zstd-jni-1.5.2-1.jar" you added. I think you can remove this and try again.

Thanks.

**Comment by yuzq24** [comment]
> > > Hi, @1141086028 , please check the failed check step "Backend / Dependency licenses" , it seems a missing dependency jar is needed https://github.com/apache/incubator-seatunnel/runs/7382295880?check_suite_focus=true
> > 
> > 
> > Hi, @legendtkl, I checked the error log information and found that "org.apache.seatunnel:connector-common:jar:2.1.3-SNAPSHOT" was missing, but I didn't reference this jar. I didn't know how to solve this problem
> 
> Hi, @1141086028 , I checked the error log. The failure is caused by the jar "zstd-jni-1.5.2-1.jar" you added. I think you can remove this and try again.
> 
> Thanks.

Hi,@legendtkl, I have removed  the jar "zstd-jni-1.5.2-1.jar" . Please check it again. 
Thank you.

**Comment by legendtkl** [comment]
@Hisoka-X  would you help to approve running workflow? It seems the workflow is blocked by Github mechanism.

image

**Comment by Hisoka-X** [comment]
Some CI problem will be fixed in #2242. After that PR merged, then CI can check dependency license normally.

**Comment by legendtkl** [comment]
hi, @1141086028 , this PR need to rebase the dev branch to resolve the failed check.

**Comment by yuzq24** [comment]
> hi, @1141086028 , this PR need to rebase the dev branch to resolve the failed check.

hi, @legendtkl , I have finished rebase operation, please check again. thanks!

**Comment by EricJoy2048** [comment]
Hi, what's news about this pr?

**Comment by EricJoy2048** [comment]
I will close this pr because it has not been updated for too long. If necessary, please reopen it.

**Pull request closed without merging**